### DISCUSSION TITLE
Extend to use "Corporate" Patterns / Modules ?

### DISCUSSION THREAD

**Discussion by andyjballgit** [microsoft/NubesGen#352] [Ideas]
Hi there, 

Just found this project , looks really interesting / useful. 

I hope this isnt there already ... But a great extension / feature would be have it reference a repository of custom Terraform modules. 

The use case here is in Corporate / Hybrid environments where are various security requirements (Storage Account must have Private Endpoint) and infrastructure requirements (DNS / Firewall rules)

cheers
Andy.

**Comment by cmaneu** [comment]
Hello @andyjballgit,
Thanks for this comment. Your idea is partially tracked by #364. For NubesGen v2, we plan to have a "repository-like" structure where you can provide your own repository of modules. You can watch #375 for progress ;)
